MEvoLibGUI/MEvoLibGUI/settings.py

```diff
@@ -0,0 +1,127 @@
+"""
+Django settings for MEvoLibGUI project.
+
+Generated by 'django-admin startproject' using Django 4.2.7.
+
+For more information on this file, see
+https://docs.djangoproject.com/en/4.2/topics/settings/
+
+For the full list of settings and their values, see
+https://docs.djangoproject.com/en/4.2/ref/settings/
+"""
+
+from pathlib import Path
+
+
+# Build paths inside the project like this: BASE_DIR / 'subdir'.
+BASE_DIR = Path(__file__).parents[1]
+
+# Quick-start development settings - unsuitable for production
+# See https://docs.djangoproject.com/en/4.2/howto/deployment/checklist/
+
+# SECURITY WARNING: keep the secret key used in production secret!
+SECRET_KEY = "django-insecure-&h=kw#*d*@zf+dr%knpgx6b*hxys$m5@1pw)cfxm028^(*1$-3"

```

Comment:
Be careful of commiting secret keys! Load secret keys from files, or encrypt it.

Reply:
I agree, but how could a user that wants to use the GUI work without it?
Directly or not, you have to give it to them, so I was (and I'm still) unsure on how to do it safely.